src/utils.js

```diff
@@ -74,7 +74,16 @@ export function getNormalizedAndLocalizedLanguageDefinitions( editor ) {
  * @param {Object.<String,String>}
  */
 export function getPropertyAssociation( languageDefs, key, value ) {
-	return Object.assign( {}, ...languageDefs.map( def => ( { [ def[ key ] ]: def[ value ] } ) ) );
+	return Object.assign( {}, ...languageDefs.map( def => {
```

Comment:
You know what... the longer I read this helper the more I think I overcomplicated it :P

Let's maybe:

```js
export function getPropertyAssociation( languageDefs, key, value ) {
	const association = {};

	for ( const def of languageDefs ) {
		if ( key === 'class' ) {
			association[ def[ key ].split( ' ' ).shift() ] = def[ value ];
		} else {
			association[ def[ key ] ] = def[ value ];
		}
	}

	return association;
}
```

WDYT?

BTW: The `indexOf` check is not necessary because `split` always returns something and since the method is called for `class` once per editor lifecycle, we don't need to optimize it much.

Reply:
I'm fine with that change 👌